zulip_bots/zulip_bots/bots/googlesearch/doc.md

```diff
@@ -4,6 +4,11 @@ This bot allows users to do Google search queries and have the bot
 respond with the first search result.  It is by default set to the
 highest safe-search setting.
 
+## Setup
+
+This bot uses the google python package.
+Run `pip install google` to install.
```

Comment:
You might want to remove this line now that we have added this to the provision script.

Reply:
@fredfishgames this seems outdated.